### DISCUSSION TITLE
blog/posts/230211-gatsby-8-comments/

### DISCUSSION THREAD

**Discussion by giscus** [ricale/blog#74] [Giscus]
# blog/posts/230211-gatsby-8-comments/

이전 글들에서는 Gatsby 기본 기능을 구현을 위한 튜토리얼을 소개하며 태그 기능을 구현했고, MDX 글 안에 인라인 이미지와 테이블을 삽입했으며, 시리즈 기능도 만들어보았고, 임시글 기능도 구현했고, Google Analytics 도 적용해보았으며…

https://ricale.kr/blog/posts/230211-gatsby-8-comments/

**Comment by Byongho96** [comment]
글 잘 봤어요!! 디테일한 부분까지 잘 정리해주셨네요

**Reply by ricale** [reply]
감사합니다 😊
